includes/wikia/models/tests/DesignSystemSharedLinksTest.php

```diff
@@ -113,6 +122,36 @@ public function getHrefDataProvider() {
 				],
 				null
 			],
+			[
+				WIKIA_ENV_PREVIEW,
+				'en',
+				[
+					'en' => [
+						'create-new-wiki' => 'http://fallout.wikia.com/wiki/test'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "http://fallout.wikia.com/wiki/test" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)